src/map/script.cpp

```diff
@@ -7566,7 +7568,7 @@ BUILDIN_FUNC(getitem2)
 	} else {
 		nameid = script_getnum(st, 2);
 		if( (item_data = itemdb_exists(nameid)) == NULL ){
-			ShowError("buildin_getitem2: Nonexistant item %d requested (by conv_num).\n", nameid);
+			ShowError("buildin_getitem2: Nonexistant item %u requested (by conv_num).\n", nameid);
```

Comment:
Missing change for BUILDIN_FUNC(getitem2)

	uint32 nameid;
	unsigned short amount;